setup/k8s/vectordb.yaml

```diff
@@ -61,30 +46,58 @@ spec:
               cpu: "500m"
               memory: "4Gi"
             limits:
-              cpu: "1"
+              cpu: "500m"
```

Comment:
Same comment re: limits. The vector DB in particular can sometimes consume a lot of CPU, when adding a lot of documents to an indexed table (laws refresh). Not critical to address right now though. I expect autoscaling will not work until we have increased vCPU available for the node pool.

Reply:
Same response. Also, increased the quota and used ScalingCalculator to find a good balance.